arm-compute/2016-03-30/swagger/compute.json

```diff
@@ -1171,7 +1171,7 @@
           }
         },
         "x-ms-pageable": {
-          "nextLinkName": null
+          "nextLinkName": "nextLink"
         }
       }
```

Comment:
This change looks fine but I see that @hyonholee has made the same change in  PR #359 and is also trying to fix #352 to fix the broken CI.  If he's successful this PR can be cancelled